src/Groups/GAPGroups.jl

```diff
@@ -259,8 +259,8 @@ end
 """
     perm(G::PermGroup, L::AbstractVector{<:Integer})
     (G::PermGroup)(L::AbstractVector{<:Integer})
-Return the permutation `x` sending `i` into `L[i]` for every `i`. `L` must contain exactly one time every integer from 1 to n for n = length(`L`).
-The parent of `x` is `G`. If `x` is not in `G`, return ERROR.
+Return the permutation `x` sending `i` into `L[i]` for every `i`. `L` must contain exactly one time every integer from 1 to `n` for `n` = length(`L`).
+The parent of `x` is `G`. If `x` is not contained in `G`, an ERROR is returned. For `gap_perm`, the parent group of `x` is set as Sym(`n`), where `n` is the largest moved point of `x`.
```

Comment:
Are you sure that an "ERROR is returned"?  Isn't rather a exception thrown? Same for other sections with similar text

Reply:
For the permutation groups, an `ArgumentError` is returned. I'll check for similar parts.